src/vm/peimage.cpp

```diff
@@ -969,7 +969,7 @@ PTR_PEImageLayout PEImage::CreateLayoutMapped()
     {
         // For CoreCLR, try to load all files via LoadLibrary first. If LoadLibrary did not work, retry using 
```

Comment:
nit: Please update these comments to align with code

Reply:
Added more comments on the throw